### ISSUE TITLE
video-js-swf w/t flashls does not play video to the very end.

### ISSUE THREAD

**Issue by bparadie** [mangui/flashls#437]
This issue might be related to https://github.com/mangui/flashls/issues/119 .
#### Steps
- 1. Open http://flashls-flop.rhcloud.com/6J2CDiFet5E/master.m3u8 in Safari and let the video play to the end:

<img width="429" alt="screen shot 2015-11-13 at 1 14 25 pm" src="https://cloud.githubusercontent.com/assets/1578115/11157802/85a980cc-8a08-11e5-8a1f-56e95dc555f9.png">
- 2. Go to http://www.flashls.org/videojs/flash_demo.html and load and play http://flashls-flop.rhcloud.com/6J2CDiFet5E/master.m3u8 as the selected custom HLS playlist to the very end: 

![screen shot 2015-11-11 at 2 42 04 pm](https://cloud.githubusercontent.com/assets/1578115/11157821/aed0d810-8a08-11e5-9d3b-6d65068c3aa3.png)
- 3. Move the playhead to the start of the video:

![screen shot 2015-11-11 at 2 42 17 pm](https://cloud.githubusercontent.com/assets/1578115/11157835/c119aa6a-8a08-11e5-968a-cd6362a34fd3.png)
- 4. Move the playhead to the end of the video:

![screen shot 2015-11-11 at 2 42 27 pm](https://cloud.githubusercontent.com/assets/1578115/11157847/d35b20dc-8a08-11e5-8f49-32fa6e7123bb.png)
- 5. Hit the play button: 

![screen shot 2015-11-11 at 2 42 39 pm](https://cloud.githubusercontent.com/assets/1578115/11157866/ece9b04a-8a08-11e5-9491-193efdb6c2ca.png)
#### Results

_Step 1_: This is our reference example.

_Step 2_: Now, in this example I could already see that we didn't reach the last frame. But that doesn't always happen. Note that the end is marked at `0:02` and the rope is not straight as in the Safari example.

_Step 5_: The video starts from the beginning. But this time it always stops before the very end. The rope is not straight as in the Safari example. Note that the progress bar now also shows `0:03` as the duration for the clip and not `0:02` as after step 1.

**Comment by mangui** [comment]
http://www.flashls.org/videojs/flash_demo.html is outdated.
Could you check if this is reproducible with latest chromeless player ?

**Comment by bparadie** [comment]
@mangui: I did regress this bug with the latest chrome less player. It is showing the same results.

**Comment by bparadie** [comment]
@mangui: Do you need any extra information?

**Comment by mangui** [comment]
no, i need time ...

**Comment by mangui** [comment]
@bparadie looks like http://flashls-flop.rhcloud.com/6J2CDiFet5E/master.m3u8 is down ?

**Comment by bparadie** [comment]
@mangui Yep, it's down for some reason. Let me check this weekend. Thx!

**Comment by bparadie** [comment]
@mangui I've restarted the node.js app and it works for me now:

``` bash
wget http://flashls-flop.rhcloud.com/6J2CDiFet5E/master.m3u8 
--2015-12-06 11:39:59--  http://flashls-flop.rhcloud.com/6J2CDiFet5E/master.m3u8
Resolving flashls-flop.rhcloud.com... 54.210.166.217
Connecting to flashls-flop.rhcloud.com|54.210.166.217|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 443 [application/x-mpegurl]
Saving to: 'master.m3u8'

master.m3u8                          100%[======================================================================>]     443  --.-KB/s   in 0s     

2015-12-06 11:39:59 (32.5 MB/s) - 'master.m3u8' saved [443/443]
```

If you use a browser to check this site  make sure you use Safari not Chrome.
This link shows the video:
http://flashls-flop.rhcloud.com/

**Comment by mangui** [comment]
it is working fine with latest chromeless player on OSX/Chrome and OSX/FF
http://www.flashls.org/latest/examples/chromeless/?src=http%3A%2F%2Fflashls-flop.rhcloud.com%2F6J2CDiFet5E%2Fmaster.m3u8

**Comment by bparadie** [comment]
@mangui: The bug is definitely not fixed. Although the behavior has improved. Here is what I am seeing: The video plays until it reaches the end. In older versions of `flashls` the video would just stop and the last displayed frame was with a curved rope.
In the latest version of `flashls` the video still stops and does not play the last second or so. But at one point the player catches up and ends up always displaying the last frame.

Here is a GIF generated from my screen (my apologies for the bad quality):

![flashls-bug](https://cloud.githubusercontent.com/assets/1578115/11725892/d4f236f0-9f30-11e5-9fbc-c94a9496cc89.gif)

The bottom line is: 
- The bug has not been fixed  in the latest version of `flashls`
- the experience has been improved by displaying the last frame.

You might have to try playing the video a few times in order to reproduce it.

**Comment by mangui** [comment]
I tried a couple of times and I always see the last second.
with http://whatsmybrowser.org/b/1FHA2C3
and http://whatsmybrowser.org/b/M8DR0KP

**Comment by bparadie** [comment]
@mangui Unfortunately `whatsmybrowser` is not showing the Chrome version. 
We can reproduce this problem easily on OSX 10.10.4 in Chrome Version 47.0.2526.80 (64-bit).
May I ask you to install http://www.gifgrabber.com/ and record what you are seeing on screen?
